recipes/fire-hpp/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "0.2.0-pre.20201031":
```

Comment:
```suggestion
```

Add 0.1 version instead.

Reply:
0.1 is not in a packageable state. The API has already been broken in a fundamental way and it isn't worth it.

Reply:
This is not an official pre-release, right?

Please use our `cci.<date-of-commit>` format for a version then. See also https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

Reply:
Replaced with new 0.2 release.